app/views/topics/show.html.erb

```diff
@@ -1,11 +1,12 @@
 <div class="row">
   <div class="view-lesson-planner">
     <% if Flipper.enabled?(:lesson_plans) %>
-      <%= link_to "View Lesson Planner",  current_lesson_plan_path,
-        class: "lesson-planner-link" %>
-        <span class="lesson-arrow"></span>
-        <% end %>
+      <%= link_to "View Lesson Planner", current_lesson_plan_path,
+                                         class: "lesson-planner-link" %>
```

Comment:
nit: why align with the second param? why not the first, or just indent?

Reply:
:man_shrugging: The first param is what's visible in the UI so I like to shove everything else over to the right. I don't feel strongly about it though, IMO making erb look good is a losing battle.